### DISCUSSION TITLE
Meeting Updates - Address not found

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1263] [General]
Hi there,

The Al-Anon Family Groups Australia team are making updates to their meetings (as per the updates members have sent them on the website). 

But we've come across a bug, stopping addresses from being added to the meeting - the map isn't loading the address and therefore the meeting is unable to save.

Your advise on how to fix this would be appreciated.

![Screen Shot 2023-11-14 at 11 57 18 am](https://github.com/code4recovery/12-step-meeting-list/assets/23748349/6c436099-489b-4af2-8f58-79a2b69c26d3)

**Comment by joshreisner** [comment]
this looks like a geocoding issue, sorry, we're looking into it!

**Reply by nataliefaith2** [reply]
Oh amazing, thank you!

**Reply by tim-rohrer** [reply]
@nataliefaith2, please try again. Geocoding should be working now. Please let us know if  you still have problems.

Tim

**Reply by nataliefaith2** [reply]
Fantastic - thank you @tim-rohrer and @joshreisner

**Comment by nataliefaith2** [comment]
I've just tested an online meeting and having the same issue.

It looks like the "New Meeting" button has changed to "Add a New Post" - Could an update here be causing this issue? Below is a screenshot of my dashboard.
![Screen Shot 2023-11-14 at 12 14 30 pm](https://github.com/code4recovery/12-step-meeting-list/assets/23748349/49db983f-3989-40fe-8566-e97b7be5ac8b)
